iac-extensions/kubernetes/src/test/java/org/sonar/iac/helm/HelmEvaluatorTest.java

```diff
@@ -66,14 +66,29 @@ static void cleanup() throws IOException {
   }
 
   @Test
-  void shouldThrowIfGoBinaryReturnsNonZero() {
+  void shouldThrowIfGoBinaryRejectsEmptyInput() {
     Assertions.assertThatThrownBy(() -> helmEvaluator.evaluateTemplate("/foo/bar/baz.yaml", "", ""))
       .isInstanceOf(IllegalStateException.class)
-      .hasMessage("sonar-helm-for-iac exited with non-zero exit code: 1");
+      .hasMessage("Evaluation error in Go library: read input: won't read 0 lines");
 
     Assertions.assertThat(logTester.logs(Level.DEBUG))
-      .contains("[sonar-helm-for-iac] Skipping request to read 0 lines")
-      .contains("[sonar-helm-for-iac] Expected 2 files, received 3 (values.yaml missing?)");
+      .contains("[sonar-helm-for-iac] Reading 0 lines from stdin");
+  }
+
+  @Test
```

Comment:
Unrelated to this line, but to the HelmEvaluator.
I would change `Empty evaluation result (serialization failed?)` to
`Empty evaluation result received, possible serialization failure in Go library`

Reply:
Yes, good suggestion. Actually, now in case serialization fails go binary should exit with code 1, so this branch covers unexpected empty results. I'll adjust the message accordingly